setup.py

```diff
@@ -11,19 +11,25 @@
 import sys
 
 
+DEBUGPY_BUNDLING_DISABLED = bool(os.getenv('DEBUGPY_BUNDLING_DISABLED'))
```

Comment:
For clarity, let's invert the meaning - i.e. `BUNDLE_DEBUGPY` which defaults to true but can be overridden.